acceptance/tests/resource/exec/accept_array_commands.rb

```diff
@@ -0,0 +1,26 @@
+test_name "Be able to execute array commands" do
+  tag 'audit:high',
+      'audit:acceptance'
+
+  agents.each do |agent|
+    if agent.platform =~ /windows/
+      cmd = ['cmd.exe', '/c', 'echo', '*']
+      path = 'C:\Windows\System32'
+    else
+      cmd = ['/bin/echo', '*']
+      path = '/usr/bin'
```

Comment:
If the command path is fully qualified you don't need to specify `path`, or maybe you wanted `cwd` to make the command execute in that directory?

Reply:
or maybe you can make the Windows command fully-qualified so you don't need `path` at all on line 17

Reply:
One trick is to use the `path` fact:

```puppet
exec { '...':
  ...
  path => '$::path'
}
```